service/src/main/java/bio/terra/profile/app/common/MdcHook.java

```diff
@@ -114,7 +113,7 @@ private String serializeMdc(@Nullable Map<String, String> mdcMap) {
 
   private Map<String, String> deserializeMdc(@Nullable String serializedMdc) {
     if (serializedMdc == null) {
-      return ImmutableMap.of();
```

Comment:
I'm not convinced this is a code smell, but I'm also used to FP conventions were data is generally immutable by default.

Reply:
Map.of() gives back immutablemaps, and this reduces the dependency on guava and the need to know two APIs 

https://docs.oracle.com/javase/9/docs/api/java/util/Map.html#of--
https://sonarcloud.io/project/issues?resolved=false&id=DataBiosphere_terra-billing-profile-manager&open=AYLbaQ67IfTdJA_2ktf1&tab=why

Reply:
Oh: didn't realize it was a dependency thing.